conjur/util/credentials_from_file.py

```diff
@@ -89,19 +90,28 @@ def update_api_key_entry(self, user_to_update, credential_data, new_api_key,
         hosts[credential_data['machine']] = (user_to_update, None, new_api_key)
         self.build_netrc(netrc_obj)
 
-    def remove_credentials(self, conjurrc_conjur_url):
+    def remove_credentials(self, conjurrc):
         """
         Method that removes the described login entry from netrc
         """
-        netrc_data = self.load(conjurrc_conjur_url)
+        logging.debug(f"Attempting to remove credentials from '{DEFAULT_NETRC_FILE}'")
+        # pylint: disable=no-else-return
+        if not os.path.exists(DEFAULT_NETRC_FILE):
+            return
```

Comment:
I suggest log.debug that we are trying to delete from empty file (unless this is a normal behaviour)